Applications/DataExplorer/DataView/DiagramView/DiagramList.h

```diff
@@ -142,6 +146,9 @@ class DiagramList
     /// Returns the width of the bounding box of all data points within the list.
     double width() const { return _maxX - _minX; }
 
+    /// Converts string into QDateTime-format
+    static QDateTime getDateTime(QString s);
```

Comment:
What do you mean by "free function"? Put it in BaseLib? Leave it there but take it of DiagramList?  
(In a way this is fairly specific for this class because I'm just checking a few date prepresentations. This is by no means a comprehensive, super-safe, multi-purpose function. Just saying.)

Reply:
A free function is not a member function. If the function is specific to the translation unit (DiagramList.cpp in this case) add the function to the cpp file only, there no need to add it to the class interface.

Reply:
I need the method in both DiagramList and DiagramPrefsDialog. What's your preferred way to implement this then? As a free function in both files? A seperate file?